### PR TITLE
Add initial dataviz for strategy allocations

### PR DESCRIPTION
**Pull request by theecrit**
Addresses (does not close) #68.

I _think_ I've resolved the conflicts I was seeing and haven't introduced any other weird changes, but please check my work! The chart colors need to be revised for accessibility and we probably don't want to stick with Infogram embeds but it was the quickest way for me to get something up there. :) 

Screenshot 2022-02-10 at 21-20-37 About Measure Z

### CODE REVIEW

**Comment by aemann2** [comment]
Seeing some errors in the console when I run this in dev:
Screen Shot 2022-02-11 at 2 13 48 PM

I'm going to try to figure out where these are coming from

File: src/about-measure-z.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Measure Z funds are distributed in the following ways:
 
 ### How much money is spent
 
-_[placeholder for dataviz]_
+{% include "dataviz-strategy-spending.njk" %}
```
Comment (aemann2) [inline src/about-measure-z.md:44]: Liquid didn't like this for some reason, so I removed it, took the data-viz code out of `dataviz-strategy-spending.njk`, and pasted it directly into the `about-measure-z.md` page. Now it's working and everything is building again.

**Comment by theecrit** [comment]
@aemann2 I went ahead and got this branch updated with `main`, then tried again with the include — and it seems to be building properly now that everything's been updated. Want to give it another look for me? I'd love to stick with includes if possible because it's a little more manageable if we want to move visualizations around as we continue to iterate.

**Comment by aemann2** [comment]
@theecrit sounds good, I'll look at this in the next couple of days and will merge if all's good!

**Pull request merged**